diff --git a/Makefile b/Makefile
index b2bbd21..c49790c 100644
--- a/Makefile
+++ b/Makefile
@@ -13,7 +13,7 @@ nohbm:
 hbm:
 	mkdir -p synthesis_results_HBM
 	cp Makefile.synthesis synthesis_results_HBM/Makefile
-	make -C synthesis_results_HBM FPGAPART=xcu280-fsvh2892-2L-e -j4
+	make -C synthesis_results_HBM FPGAPART=xczu48dr-ffvg1517-2-e -j4
 
 .PHONY:versalaicore
 versalaicore:
diff --git a/Makefile.synthesis b/Makefile.synthesis
index ad0d4af..6ee0095 100644
--- a/Makefile.synthesis
+++ b/Makefile.synthesis
@@ -12,7 +12,7 @@ PORTSRC=$(TOPDIR)/hls/port_handler
 TCLDIR=$(TOPDIR)/scripts
 
 
-FPGAPART ?= xcu250-figd2104-2L-e
+FPGAPART ?= xczu48dr-ffvg1517-2-e
 
 project = ARP_hls_prj ETH_inserter_hls_prj ICMP_hls_prj PKT_HANDLER_prj UDP_prj
 
